terraspace/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+gem "terraspace", '~> 1.1.7'
```

Comment:
Shame we didn't use the latest version of terraspace for this

Reply:
Yeaa.. You're right! Didn't think about it before.
I don't know how much effort would it be to upgrade, but I could give it a shot in a next PR